builder/build.go

```diff
@@ -128,3 +137,62 @@ func ensureHandlerPath(handler string) error {
 
 	return nil
 }
+
+// TODO How will this be used with another build_options in future? Should it be renamed?
+func getBuildOptionPackages(buildOption string, language string) (string, error) {
+	var err error
+	result := ""
+
+	if buildOption != "" {
+		buildOptions, err := deriveBuildOptions(language)
+
+		if err != nil {
+			return result, err
+		}
+
+		switch buildOption {
+		case "dev":
```

Comment:
We cant have a case/select that covers each buildOption as we'll have to open the code on every template change and 3rd party templates will be constrained to officially supported options.